x-pack/plugins/fleet/server/services/epm/packages/install.ts

```diff
@@ -110,10 +112,20 @@ export async function ensureInstalledPackage(options: {
     spaceId = DEFAULT_SPACE_ID,
   } = options;
 
+  let prerelease: boolean = false;
+  try {
+    // check prerelease setting to enable installing latest GA version, even if there is a newer prerelease version
+    ({ prerelease_integrations_enabled: prerelease } = await getSettings(savedObjectsClient));
+  } catch (err) {
+    appContextService
+      .getLogger()
+      .warn('Error while trying to load prerelease flag from settings, defaulting to false', err);
+  }
+
```

Comment:
nit: Maybe worth refactoring that block in his own method I saw this a few time.

Reply:
done